sandcastle/api.py

```diff
@@ -322,7 +322,10 @@ def create_pod(self, pod_manifest: Dict) -> Dict:
 
         :return: response from the API server
         """
-        # if we hit timebound quota, let's try 5 times with expo backoff
+        # if we hit timebound quota, let's try RETRY_CREATE_POD_MAX times with expo backoff
+        # 2 ** 7 = 128 = 2 minutes
+        # 2 ** 8 = 256 = 4 minutes
+        # in total we try for ~8 minutes
```

Comment:
Better to provide late result than nothing I think.

Reply:
agreed, that was exactly my thinking